README.md

```diff
@@ -1,70 +1,76 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Project Name
 
-## Available Scripts
+> One paragraph statement about the project.
 
-In the project directory, you can run:
+![screenshot](./app_screenshot.png)
 
-### `yarn start`
+Video presentation [here](https://www.loom.com/share/66e7981f5cd143bb92b3766f13409b53)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Built With
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+- Major languages
+- Frameworks
+- Technologies used
 
-### `yarn test`
+## Live Demo
 
```

Comment:
- [ ] Kindly ensure that the app is deployed and the live link is provided in the readme as per the project requirments.